src/credentials/CHIPCert.cpp

```diff
@@ -677,15 +681,15 @@ CHIP_ERROR ChipDN::GetCertFabricId(uint64_t & fabricId) const
 {
     uint8_t rdnCount = RDNCount();
 
-    fabricId = UINT64_MAX;
+    fabricId = kUndefinedFabricIndex;
 
     for (uint8_t i = 0; i < rdnCount; i++)
     {
         switch (rdn[i].mAttrOID)
         {
         case kOID_AttributeType_ChipFabricId:
-            // Ensure only one FabricID RDN present, since start value is UINT64_MAX, which is reserved and never seen.
-            VerifyOrReturnError(fabricId == UINT64_MAX, CHIP_ERROR_WRONG_CERT_TYPE);
+            // Ensure only one FabricID RDN present, since start value is kUndefinedFabricIndex, which is reserved and never seen.
+            VerifyOrReturnError(fabricId == kUndefinedFabricIndex, CHIP_ERROR_WRONG_CERT_TYPE);
```

Comment:
Again, this should not be using fabric indices.  Should be IsValidFabricId?

Reply:
fixed